src/rust/bitbox02/src/memory.rs

```diff
@@ -16,16 +16,16 @@
 // deduct one for the null terminator.
 pub const DEVICE_NAME_MAX_LEN: usize = bitbox02_sys::MEMORY_DEVICE_NAME_MAX_LEN as usize - 1;
 
-/// `name.as_bytes()` must be smaller or equal to
-/// `DEVICE_NAME_MAX_LEN`, otherwise this function panics.
+/// `name.as_bytes()` must be smaller or equal to `DEVICE_NAME_MAX_LEN`.
```

Comment:
not sure this change helps, as now the caller doesn't know if it is a memory or invalid_input error. imho it's fine to document the panic and have the caller check the input, which you do now anyway.

Reply:
> now the caller doesn't know if it is a memory or invalid_input 

the caller didn't know it before either. it was a panic on invalid input

> it's fine to document the panic

but it doesn't panic anymore.

> have the caller check the input, which you do now anyway.

true. so, I'm thinking to revert this part since the input is now validated in the API. sounds good?

Reply:
reverted.